app/vmagent/main.go

```diff
@@ -343,7 +343,7 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 		w.WriteHeader(202)
 		fmt.Fprintf(w, `{"status":"ok"}`)
 		return true
-	case "/datadog/api/v1/series":
+	case "/datadog/api/v1/series", "/datadog/api/v2/series", "/datadog/api/beta/sketches":
```

Comment:
there is also `processMultitenantRequest` func to be updated

Reply:
And I think we should update `datadogWriteRequests` metric to account for different API calls (add new labels)

Reply:
I'd vote to split methods to different handlers instead of using map.
However, I noticed that for `influx` protocol @valyala didn't add version notion at all.
I personally would prefer having label to reflect the exact path for request: was it v1 or v2 or sketches.